ci-operator/config/openshift/lightspeed-operator/openshift-lightspeed-operator-main.yaml

```diff
@@ -68,14 +68,67 @@ tests:
   container:
     clone: true
     from: golangci-lint
-- as: bundle-e2e
+- as: bundle-e2e-4-15
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    labels:
+      region: us-east-2
+    owner: obs-logging
```

Comment:
did you talk to the logging team about sharing their cluster pool?  Do we need to reconsider the size of it if we're going to share one?

cc @JoaoFula who was going to setup clusterpools for OLS

Reply:
I'm all for getting more clusters added to the pool for us. My impression is that it's not the size of the cluster, it's the number of clusters in the pool, if we're going to ask for the same, say, 4.16 cluster as Logging. I didn't talk to the Logging team yet, but in the course of https://redhat-internal.slack.com/archives/CBN38N3MW/p1718380798017079 the DPTP folks didn't raise it as a concern. FWIW, there's only one 4.16 cluster in the pool according to https://docs.ci.openshift.org/docs/how-tos/cluster-claim/#existing-cluster-pools.

Reply:
> My impression is that it's not the size of the cluster, it's the number of clusters in the pool,

sorry if i was unclear, but yes "size of the pool" is what i was referring to needing to look at if we're going to share the pool w/ other teams.

Reply:
> FWIW, there's only one 4.16 cluster in the pool according to https://docs.ci.openshift.org/docs/how-tos/cluster-claim/#existing-cluster-pools.

the 4.16 pool size is 5:
https://github.com/openshift/release/blob/9e1aa8c6a36f7ebcefba6051369994344d51b287/clusters/hosted-mgmt/hive/pools/openshift-observability/obs-logging-ocp-4-16-0-amd64-aws-us-east-1_clusterpool.yaml#L27

the 4.15 pool size is 10:
https://github.com/openshift/release/blob/9e1aa8c6a36f7ebcefba6051369994344d51b287/clusters/hosted-mgmt/hive/pools/openshift-observability/obs-logging-ocp-4-15-0-amd64-aws-us-east-2_clusterpool.yaml#L27

and btw i'm not sure if max size represents a limit on how many clusters can be idle in the pool, or how many can be in use (by running jobs) simultaneously.  If it's the latter, it's even more of a concern if we're sharing it w/ another team.

(running out of idle clusters isn't a big deal, it just means the job takes longer to run because it has to provision a new cluster from scratch, but if we can't even request a cluster because we hit the max size, that will delay jobs much longer).

lastly if you look at:
https://github.com/openshift/release/tree/master/clusters/hosted-mgmt/hive/pools/openshift-observability

you can see that every other team appears to have created our own pool, so i don't think we should be reusing another team's pool rather than creating our own.  

Among other things we don't want to have to worry what happens if the logging team decides for some reason to change the configuration of their pool in a way that breaks our OLS jobs.

Reply:
> Hive will fulfill all claims until the number of living clusters reaches the pool’s maxSize

so it's as a i suspected, if the pool max size is 5, then only 5 simultaneous CI jobs(assuming they are all claiming clusters from the same pool) would be able to run at the same time and any additional jobs would have to wait to start.

Reply:
ok, so I'll start the task of creating the cluster pool tomorrow morning and try to have it ready then. Was already discussing it with the folks from DVO that were doing that migration and it should be quite easy (just a few parameters need changing).

Reply:
@JoaoFula looks like we'll need a 4.15 and a 4.16 pool

Reply:
Thanks, @JoaoFula ! Just give me a shout and I'll plug in the new pool details into this PR and https://github.com/openshift/release/pull/53258.

Reply:
owner is

`ols`